Aditi/aditi_first_assignment/index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html>
+<head>
+     <title>css demo</title>
+     <link rel="stylesheet"href="style.css"/>
+</head>
+ <body>
+    <section class="header-section">
+	<header>
+	<hgroup class="website-img">
+	   <img src="fullstacklogo.PNG"alt="website logo"height=50px/>
```

Comment:
remove this height and add to img tag in css.

Reply:
done